src/Files.App.Controls/BreadcrumbBar/BreadcrumbBarLayout.cs

```diff
@@ -30,30 +30,40 @@ public BreadcrumbBarLayout(BreadcrumbBar breadcrumb)
 
 		protected override Size MeasureOverride(NonVirtualizingLayoutContext context, Size availableSize)
 		{
-			var accumulatedSize = new Size(0, 0);
+			// Early exit for null or empty context
+			if (context?.Children == null || context.Children.Count == 0)
+			{
+				return new Size(0, 0);
+			}
```

Comment:
```suggestion
			if (context?.Children is null || context.Children.Count is 0)
				return new Size(0, 0);
```

Reply:
@MohammadSamiIslam can you please review? Thanks!